blueprints/cloud-operations/network-dashboard/cloud-function/main.py

```diff
@@ -12,79 +12,82 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+# CFv2 define whether to use Cloud function 2nd generation or 1st generation
 
-from distutils.command.config import config
```

Comment:
I think your linter changed the formatting of all the python code. We use yapf with `{based_on_style: google, indent_width: 2, SPLIT_BEFORE_NAMED_ASSIGNS: false}`. Can you update the formating?

Reply:
I have formatted the python code